tinyagentos/routes/decisions.py

```
@@ -244,5 +246,33 @@ async def answer_decision(decision_id: str, body: AnswerIn, request: Request, us
+    if grants is None or not app_id:
+        return
+    user_id = decision.get("user_id") or ""
+    granted = set(value if isinstance(value, list) else [value])
+    try:
+        for cap in caps:
+            await grants.set_decision(
+                user_id, app_id, cap,
+                decision="granted" if cap in granted else "denied",
+            )
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)